go/vt/dbconnpool/connection.go

```diff
@@ -36,24 +37,21 @@ func (dbc *DBConnection) handleError(err error) {
 
 // ExecuteFetch is part of PoolConnection interface.
 func (dbc *DBConnection) ExecuteFetch(query string, maxrows int, wantfields bool) (*proto.QueryResult, error) {
-	start := time.Now()
+	defer dbc.mysqlStats.Record("Exec", time.Now())
```

Comment:
"""
Each time a "defer" statement executes, the function value and parameters to the call are evaluated as usual and saved anew but the actual function is not invoked.
"""
https://golang.org/ref/spec#Defer_statements